src/samples/BotToBot/Bot1/.config/dotnet-tools.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
Noise?

Reply:
I just did a push removing this and added to .gitignore